fe/fe-core/src/main/java/com/starrocks/alter/SchemaChangeHandler.java

```diff
@@ -491,6 +492,10 @@ private void processReorderColumn(ReorderColumnsClause alterClause, OlapTable ol
             if (!colNameSet.add(colName)) {
                 throw new DdlException("Duplicated column[" + colName + "]");
             }
+            if (oneCol.get().isKey() && keyIdx++ != targetIndexSchema.indexOf(oneCol.get())
```

Comment:
1. create table A(k1,k2,k3,v1,v2) primary key(k1, k2, k3) order by (k1, k2,k3)
    alter table A order by (v1,v2)
2. create table A primary key(k1, k2, k3,v1,v2) order by (v1, v2)
    alter table A order by (v2, v1) 
3. create table A primary key(k1, k2, k3,v1,v2) order by (k1, k2, k3)
    alter table A order by (k3, k2, k1)

Reply:
1/2 is ok, 3 is not ok?